src/app/SuccessButton.css

```diff
@@ -0,0 +1,3 @@
+.button {
+  background-color: green !important;

```

Comment:
Well, using `beta.5` the example from my first comment now works! But I still need `!important` on a pure `.button` selector.
```css
@import "react-toolbox/lib/button/theme.css";

.button {
  background-color: var(--color-black) !important;
}
```
else button is not black